probcal/enums.py

```diff
@@ -9,9 +9,18 @@ class AcceleratorType(Enum):
 
 
 class HeadType(Enum):
+    MEAN = "mean"
     GAUSSIAN = "gaussian"
     POISSON = "poisson"
+    DOUBLE_POISSON = "double_poisson"
     NEGATIVE_BINOMIAL = "negative_binomial"
+    POISSON_GLM = "poisson_glm"
+    NEGATIVE_BINOMIAL_GLM = "negative_binomial_glm"
+    DOUBLE_POISSON_GLM = "double_poisson_glm"
+    FAITHFUL_GAUSSIAN = "faithful_gaussian"
+    NATURAL_GAUSSIAN = "natural_gaussian"
+    MULTI_CLASS = "multi_class"
```

Comment:
It doesn't look like all the model code for these types of heads is ported over from `deep-uncertainty` yet. Maybe in a future change we can copy those to this repo (if we end up using them)